SECURITY.md

```diff
@@ -45,4 +45,4 @@ Once the request is executed no secrets are accessed until another request is to
 
 Another aspect to consider is logging. Logging is **disabled** by default. When enabled, logging may leak user secrets, so an appropriate logging level should be selected to only expose as much information as is secure in given context.
```

Comment:
Jirasek :D Drop "Another aspect to consider is logging"

Reply:
Heh, I haven't really checked the SECURITY document, only to fix "Superface SDK". I would like to go through it in a separate PR, ok?